src/main/kotlin/no/nav/bidrag/grunnlag/service/GrunnlagspakkeService.kt

```diff
@@ -375,12 +377,9 @@ class GrunnlagspakkeService(
     return persistenceService.hentKomplettGrunnlagspakke(grunnlagspakkeId)
   }
 
-  fun settGyldigTildatoGrunnlagspakke(settGyldigTilDatoForGrunnlagspakkeRequest: SettGyldigTilDatoForGrunnlagspakkeRequest): Int {
-    persistenceService.validerGrunnlagspakke(settGyldigTilDatoForGrunnlagspakkeRequest.grunnlagspakkeId)
-    return persistenceService.settGyldigTildatoGrunnlagspakke(
-      settGyldigTilDatoForGrunnlagspakkeRequest.grunnlagspakkeId,
-      LocalDate.parse(settGyldigTilDatoForGrunnlagspakkeRequest.gyldigTil)
-    )
+  fun lukkGrunnlagspakke(lukkGrunnlagspakkeRequest: LukkGrunnlagspakkeRequest): Int {
+    return persistenceService.lukkGrunnlagspakke(
+      lukkGrunnlagspakkeRequest.grunnlagspakkeId)
```

Comment:
Legge til `validerGrunnlagspakke` for å trigge eventuell 404 feil fremfor 500 feil

Reply:
Ok